app/scripts/metamask-controller.js

```diff
@@ -2237,6 +2237,9 @@ export default class MetamaskController extends EventEmitter {
     this.txController.startIncomingTransactionPolling();
     this.tokenDetectionController.enable();
 
+    // trigger start detecting nfts
+    this.nftDetectionController.start();
```

Comment:
By doing this in `triggerNetworkrequests`, I'm pretty sure it's going to fetch NFTs every time you open the MM popup.  Where it did not do so before.  We have to be careful this won't end up sending more requests than before.

Reply:
I've got a `resume` polling feature for that, but not ready to consume yet: https://github.com/MetaMask/core/pull/4128

We could make this change for now if we think it will reduce traffic overall.  And further reduce it with a resume feature later.

Reply:
I like that 🙏  we can get this in, monitor traffic then update with resume feature in future 🙏